### ISSUE TITLE
Translating to English (Shavian)

### ISSUE THREAD

**Issue by trosel** [syphon-org/syphon#587]
I was thinking of translating the app to the [Shavian alphabet for English](https://en.wikipedia.org/wiki/Shavian_alphabet). Weblate now supports it.

Would that translation be accepted by this team?

There is also a font that extends Inter to add more characters including Shavian: https://github.com/Shavian-info/interalia

**Comment by 0x1a8510f2** [comment]
While I can't speak for the entirety of the Syphon project, I can't imagine any translation officially supported by Weblate to be unwelcome.

Generally, any contributions to Syphon are appreciated, so if you're willing to take the time to work on this translation, please do! :D

**Comment by EdGeraghty** [comment]
Not that I speak for the team either, but the only blocker I can see here is the requirement for another font to support the glyphs.

I can't see an issue with the translation in and of itself though!

**Comment by ereio** [comment]
Correct, was going to mention this. Feel free to add the language through weblate. We'll save it there until we have time to support the glyphs, it just might take a bit in between those steps. Happy to do it though!

**Comment by trosel** [comment]
Quick Question: Is there a way to translate the strings in a file? skipping the weblate interface? it’s not super easy to find and replace all instances of the word “chats”, for example

**Comment by 0x1a8510f2** [comment]
Sure is!

All the translations are stored here: https://github.com/syphon-org/syphon/tree/dev/assets%2Ftranslations

You should be able to copy the regular English strings over to a new file and translate them there. According to the [Weblate commit](https://github.com/syphon-org/syphon/commit/3bb007c7b1733604f7e874e514076976f084878a), the file should be named `en-Shaw.json`.

You may want to wait until [#591](https://github.com/syphon-org/syphon/pull/591) is merged first though, to avoid merge conflicts.

**Comment by trosel** [comment]
Do I need to add Shavian to the dropdown language selector in the app?
